profiles/default.preseed

```diff
@@ -69,7 +69,7 @@ d-i apt-setup/services-select multiselect security, updates
 d-i apt-setup/security_host string security.debian.org
 
 d-i apt-setup/local0/repository string \
-	http://forge.yunohost.org/debian/ stretch stable
+	http://forge.yunohost.org/debian/ buster stable
```

Comment:
Where is the builder for these packages ?

Reply:
That happens in https://github.com/YunoHost/vinaigrette (which is kind of a mess ... but at least it's more organized and versionned than 2 years ago :/ )